packages/extension-browser/src/devtools/views/controls/inspect-button.tsx

```diff
@@ -16,6 +16,13 @@ const InspectButton = ({ target }: Props) => {
         // Verify target is actually a number since it originates from untrusted snapshot data.
         if (typeof target === 'number') {
             evaluate(`inspect(__webhint.findNode(${target}))`);
+            evaluate(`(function() {
```

Comment:
`inspect()` is running two times, one in each `evaluate`, is it ok?

Reply:
I just read @antross comment hehehe. I think you forgot to remove the first `evaluate`